happy/HappyState.py

```diff
@@ -200,6 +203,18 @@ def __print_node_state(self):
         node_json_data = json.dumps(node_info, sort_keys=True, indent=2)
         print node_json_data
 
+    def __print_extension_value(self):
+        """print happy extension value
+        example: weave-fabric-id
+        """
+        if self.extension is None:
+            return
+
+        print "happy topology extension {} value: ".format(self.extension)
```

Comment:
This will read a little weird.  When we pass a `--extension weave` the above will print 
    happy topology extension weave value: 
      values

Perhaps a better thing to print there would be:

    print "{} extension state:".format(self.extension)

Reply:
sure, will change it